_episodes/l1-04-abstractions.md

```diff
@@ -280,6 +277,11 @@ Now the result of `compute_a` has a hidden dependency. It's never clear whether
 calling it twice with the same input (`measurements`) will yield the same
 result.
 
+In general make sure that all variables have the most limited scope possible. If
+they're only needed within a single function define them there. Wherever
+possible make variables into constants so you know they're not being modified
```

Comment:
There are no "constants" in Python, which is what we are using in the course, so maybe  clarify this is applicable only to other languages or remove the sentence altogether.

Reply:
Maybe just change the wording like this?
```suggestion
possible treat variables as constants so you know they're not being modified
```